### ISSUE TITLE
Lsp for tsserver not working

### ISSUE THREAD

**Issue by ThatOneOcto** [NvChad/NvChad#1138]
**Describe the bug**
The language server for typescript and javascript doesn't work (I'm using tsserver), I've followed the guide [here](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#tsserver) to install tsserver and it doesn't work

**To Reproduce**
It just doesn't work don't know how to reproduce

**Expected behavior**
For LSP to work on .js and .ts files

**Screenshots**

**Desktop (please complete the following information):**
 - Gentoo
 - Alacritty
 - 0.7.0

**Additional context**

**Comment by ThatOneOcto** [comment]
I fixed it by uninstalling typescript and typescript lsp with this command `npm r -g typescript typescript-language-server` then reinstalled it with `npm i -g typescript typescript-language-server` _(This should be common sense but run it with sudo since you're installing the packages globally which goes in /usr/lib64/node_modules/ which needs root access)_

**Comment by siduck** [comment]
@ThatOneOcto lsp installer willl install tsserver for u so why install it globally? just setup tsserver  in your custom lspconfig, lspinstaller will detect it and install the lsp server

**Comment by ThatOneOcto** [comment]
I had it installed globally cause it gives me a path error also on reboot it stops working, I added `require'lspconfig'.tsserver.setup{}` in my ~/.config/nvim/lua/custom/init.lua file and it fixed it

TL;DR

Add `require'lspconfig'.tsserver.setup{}` in ~/.config/nvim/lua/custom/init.lua

**Comment by ThatOneOcto** [comment]
(Completed ticket)
